Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ButtonGroup.cpp

```diff
@@ -28,6 +29,22 @@ namespace AzToolsFramework::ViewportUi::Internal
         return m_viewportUiId;
     }
 
+    void ButtonGroup::SetDisabledButton(ButtonId buttonId, bool disabled)
+    {
+        if (auto buttonEntry = m_buttons.find(buttonId); buttonEntry != m_buttons.end())
+        {
+            switch (buttonEntry->second->m_state)
+            {
+            case Button::State::Selected:
+                ClearHighlightedButton();
+            [[fallthrough]];
```

Comment:
uber nit: Can this be move in one additional level of indentation (so we get...)

```
switch (buttonEntry->second->m_state)
            {
            case Button::State::Selected:
                ClearHighlightedButton();
                [[fallthrough]];
```
(unless that goes against the clang-format style, as then it's not worth it)